app/controllers/users_controller.rb

```diff
@@ -65,7 +65,7 @@ def create
   def create_external
     @user = User.new(params[:user])
     @user.password = generate_new_password
-    LogEvent.log(@user, :create, current_user)
+    LogEvent.log(@user, :create, @user.created_at, current_user)
```

Comment:
On that same note about timestamps, I don't think `created_at` will have been set yet because it hasn't been saved. It's possible I'm wrong.